runtime/gc_glue_java/ArrayletObjectModelBase.hpp

```diff
@@ -44,6 +44,9 @@ class GC_ArrayletObjectModelBase
 #if defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS)
 	bool _compressObjectReferences;
 #endif /* defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS) */
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+	bool _enableDoubleMapping; /** Allows arraylets to be double mapped */
```

Comment:
This variable should be initialized explicitly (to `false`). `GC_ArrayletObjectModelBase` does not have a constructor but it can be done in `GC_ArrayletObjectModelBase::initialize()`